### PR TITLE
export correctly when in electron environment

### PR DESCRIPTION
**Pull request by kareniel**
this commit adds a check to see if we're running in an electron environment and exports the browser version if it's the case. (currently electron gets the server version (an empty function)

this addresses choojs/nanocomponent#62

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ var KEY_ID = 'onloadid' + (new Date() % 9e6).toString(36)
 var KEY_ATTR = 'data-' + KEY_ID
 var INDEX = 0
 
+console.log('wut')
```
Comment (bcomnes) [inline index.js]: Extra console.log left in

File: server.js [hunk 2]
```diff
@@ -1 +1,9 @@
-module.exports = function () {}
+if (isElectron()) {
+  module.exports = require('.')
+} else {
+  module.exports = function () {}
+}
+
+function isElectron () {
+  return process && process.versions && (process.versions.electron !== undefined)
```
Comment (bcomnes) [inline server.js]: We should detect if its running in the main process or a browser window, since the main thread will not have access to the DOM.

**Review by bcomnes — CHANGES_REQUESTED** [review]

File: server.js (hunk 2 above)
Comment (bcomnes) [reply server.js]:
```js
function isElectronRenderer () { 
  return process && process.type && process.type === 'renderer'
}
```

Comment (kareniel) [reply server.js]: Ooh good point. I'll fix this during lunch. 😊

**Review by bcomnes — APPROVED** [review]

**Pull request merged**